internal/api/http/initialization/initialization.go

```diff
@@ -23,6 +23,8 @@ type Initialization struct {
 	AuthRoute    routes.AuthRoute
 	TaskRoute    routes.TaskRoute
 	SessionRoute routes.SessionRoute
+	SwaggerRoute routes.SwaggerRoute
```

Comment:
This route is unused. 

Even after adding it, it does not seem to be working, please consider removing it